src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -339,6 +341,10 @@ public Solution WithProjectDefaultNamespace(ProjectId projectId, string defaultN
         /// <summary>
         /// Creates a new solution instance with the project specified updated to have the name.
         /// </summary>
+        // TODO: decide if we want to allow "name" to be nullable. As of this writing you can pass null,
```

Comment:
💡 We need a better way to track these TODOs or they will get lost

Reply:
Now filed an issue and put it on the design review backlog.